dataclients/routestring/example_test.go

```diff
@@ -39,7 +39,4 @@ func Example() {
 	}
 
 	fmt.Println(string(content))
-
-	// Output:
-	// Hello, world!
```

Comment:
Was this on purpose or some linter recommended it?

Reply:
for some reason the tests failed here too often. It's unrelated from this PR, I should put it back.

Reply:
i see now. There is a race condition in the example and we don't know when will skipper return 404 as no routes have been configured yet, or the expected output. We can either make the test less readable by adding the necessary synchronization or we can just skip checking the output.

Reply:
fine for me